packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -241,7 +241,7 @@ export class CalloutContentBase extends React.Component<ICalloutProps, ICalloutS
             className={this._classNames.calloutMain}
             onDismiss={this.dismiss}
             onScroll={onScroll}
-            shouldRestoreFocus={true}
+            shouldRestoreFocus={!this.props.popupShouldNotRestoreFocus}
```

Comment:
Let's keep this consistent and call it "shouldRestoreFocus" rather than having a different vocabulary for different popup things. I'm guessing you added one like this so we can default to true, I'd advocate just having the default be true if it's undefined.

Reply:
Sounds good.